StepDefinition/LanguageFeatureStepDefinitions.cs

```diff
@@ -0,0 +1,101 @@
+using System;
+using NUnit.Framework;
+using NUnit.Framework.Legacy;
+using OpenQA.Selenium;
+using OpenQA.Selenium.BiDi.Modules.Log;
+using OpenQA.Selenium.Chrome;
+using Project_Mars.Pages;
+using Project_Mars.Utilities;
+using Reqnroll;
+
+
+namespace Project_Mars.StepDefinition
+{
+    [Binding]
+
+    public class LanguageFeatureStepDefinitions : CommonDriver
+    {
+        [Given("I logged into Project Mars successfully for language management")]
+        public void GivenILoggedIntoProjectMarsSuccessfullyForLanguageManagement()
+        {
+            driver = new ChromeDriver();
```

Comment:
this             driver = new ChromeDriver(); should be inplemented in the Common Driver class  
public class CommonDriver
    {
        public static IWebDriver driver;

        public void initialise()
        {
            // create driver
            driver = new ChromeDriver();

        }

    }